main.tf

```diff
@@ -17,6 +17,7 @@ module "aws-docker-mirror" {
   http_access_cidr_range                 = module.aws-networking.ip_cidr
   machine_ami                            = var.docker_mirror_machine_ami
```

Comment:
should we deprecate machine_ami in favor of this new property?

Reply:
I think it's still useful to be able to specify the exact AMI to use if needed.

Reply:
Alright!